scheduler_test.go

```diff
@@ -881,3 +881,18 @@ func TestDo(t *testing.T) {
 		})
 	}
 }
+
+func TestRemoveAfterExec(t *testing.T) {
+	s := NewScheduler(time.UTC)
+	s.StartAsync()
+
+	job, err := s.Every(1).StartAt(time.Now().Add(10*time.Second)).Do(task, s)
```

Comment:
let's set this to 1 sec so the test doesn't take as long
```suggestion
	job, err := s.Every(1).StartAt(time.Now().Add(1*time.Second)).Do(task, s)
```